src/app/extensions/store-locator/models/store/store.mapper.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+import { StoreData } from './store.interface';
+import { Store } from './store.model';
+
+@Injectable({ providedIn: 'root' })
+export class StoreMapper {
+  fromData(storeData: StoreData): Store {
+    if (storeData) {
+      return {
+        id: storeData.uuid,
+        name: storeData.name,
+        address: storeData.address,
+        city: storeData.city,
+        postalCode: storeData.postalCode,
+        country: storeData.country,
+        countryCode: storeData.countryCode,
+        email: storeData.email,
+        fax: storeData.fax,
+        phone: storeData.phoneBusiness,
+        latitude: Number(storeData.latitude),
```

Comment:
Not saying that it isn't, but are you sure using `Number` for the casting is the right tool for conversion here? 
We wouldn't want leading zeroes or any weird behavior to become a problem: [Casting Methods](https://stackoverflow.com/a/14668510/16965087)

Reply:
`Number` shouldn't throw any errors, however `parseFloat` would be more specific as coordinates are pairs of floats. I think I'll change it, so it's at least clear what kind of number we're talking about